### PR TITLE
PM4 compatibility + PHP8 compatiblity

### PR DESCRIPTION
**Pull request by alvin0319**

### CODE REVIEW

**Review by NhanAZ — COMMENTED** [review]
Maybe it's fine.

**Comment by HerryYT** [comment]
Any update?

File: src/Ifera/ScoreHud/session/PlayerManager.php [hunk 1]
```diff
@@ -33,15 +33,15 @@
...
-use pocketmine\Player;
+use pocketmine\player\Player;
 
 class PlayerManager{
 
 	/** @var PlayerSession[] */
 	private static $sessions = [];
 
 	public static function create(Player $player): void{
-		self::$sessions[$player->getRawUniqueId()] = $session = new PlayerSession($player);
```
Comment (DuoIncure) [inline src/Ifera/ScoreHud/session/PlayerManager.php:44]: `getRawUniqueId()` is superseded by `getUniqueId()->getBytes()`.  

This is seen here: https://github.com/pmmp/PocketMine-MP/blob/master/changelogs/4.0.md?#L499

**Review by DuoIncure — CHANGES_REQUESTED** [review]

File: src/Ifera/ScoreHud/session/PlayerManager.php (hunk 1 above)
Comment (Ifera) [reply src/Ifera/ScoreHud/session/PlayerManager.php:44]: I guess its fine for now, I will most likely be rewriting parts of ScoreHud and update it accordingly.

**Comment by Ifera** [comment]
Thanks for the PR!

I got really busy and didn't find time to look into this. I had planned on rewriting parts of ScoreHud for the 4.0 update of PMMP, but that will most likely be done at another time.

**Pull request merged**